### PR TITLE
User interface

### PR DESCRIPTION
**Pull request by MarvellousUbani**
The following updates were made:
- Created "user_interface" branch.
- Displayed prompts for collecting user input data.
- Created mock logic for data collection.

### CODE REVIEW

File: bin/player.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+	class Player
+        attr_reader :name, :symbol
+
+        def initialize
+            @name = gets.chomp()
+            @symbol = gets.chomp()
```
Comment (bolah2009) [inline bin/player.rb]: :bulb: Let's avoid the use of `gets` in this class. The **`main.rb` file is the only place where `puts` and `get.chomp` can be used** :blush: 

### Remember
> Add **executable file** named `main.rb` to bin directory. This file should be responsible for user interaction. This is the only one place where `puts` and `get.chomp` can be used.

File: bin/.stickler.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    fixer: true
+fixers:
+    enable: true
+files:
+  ignore:
+    - "bin/*"
+    - "db/*"
+    - "config/*"
+    - "Guardfile"
+    - "Rakefile"
```
Comment (bolah2009) [inline bin/.stickler.yml]: #### `REQUIRED`

Let's add the appropriate `.stickler.yml` file to the root of the project. Let's use [this file](https://github.com/microverseinc/linters-config/blob/master/ruby/.stickler.yml) instead. :point_down: 

``` yml
linters:
  rubocop:
    display_cop_names: true
files:
  ignore:
    - "bin/*"
    - "db/*"
    - "config/*"
    - "Guardfile"
    - "Rakefile"
```

If you are having an issue with setting up linter, read the `Troubleshooting` section of [this repo](https://github.com/microverseinc/linters-config/tree/master/ruby#troubleshooting).

### You can run the linter locally:  💥 - run `rubocop` in your local env:
>     - add `gem 'rubocop'` to `Gemfile` (not sure how to use Gemfile? Read [this](https://bundler.io/v1.15/guides/bundler_setup.html))
>     - run `bundle install`
>     - copy [.rubocop.yml](https://github.com/microverseinc/linters-config/blob/master/ruby/.rubocop.yml) to the root directory of your project
>     - run `rubocop`
>     - fix linter errors
>     - **Remember to let your Code Reviewer know that you had problems with Stickler and you used linter in local env.**

**Review by bolah2009 — CHANGES_REQUESTED** [review]
## Nice Job Team :muscle: 

### Almost there :smile: 

A few required changes
- Let's add a short summary of changes in Pull Request description, see example :point_down: 
  update-PR-description
- Linter files are added but no check on the PR. Have you [installed stickler?](https://stickler-ci.com/) :thinking: There should a `successful check` as shown in the image below. :point_down: 
  succesful check PR
- Check other comments below.

#### Kindly fix this and submit another code review request. :blush: 

Good luck with your next submission.

Cheers
[Bola](https://bolabuari.com/)

**Review by maelfosso — CHANGES_REQUESTED** [review]
Hello coders :smile: 

It's a pleasure to read a such clear code :smile: 
Nice improvement :clap: The changes required by the previous reviewers were taken in consideration :+1: 

Now, there is only one thing that is missing: **THe logic mock up**
Take a look to the description of the Milestone 2 
To remind you: You don't have a to implement the logic at that step but you have to show some user interaction about how it's going to work. Now, you collect players information. What should happen after that part? Don't you think that it's possible to show that? I mean displaying the board, asking to the current player where does he want to play, switching the players and so on, randomly say that it's draw or there is a winner?

Please, think about it and re-submit a new code review request once the changes done.
Thanks

**Review by Oghenebrume50 — APPROVED** [review]
Hi  coders, I see the changes so far were properly implemented, you can now move :smile:  this is approved :+1:

**Pull request merged**